### PR TITLE
Alex Noto

### PR DESCRIPTION
**Pull request by noto-alex-j**

### CODE REVIEW

File: lib/4_filter_numbers.js [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 module.exports = {
   filter_numbers: function(arr){
-    // Your code goes here.
+    var newarr = arr.filter(function(x){ return x < 10; })
+               .map(function(x) { return x});
```
Comment (halfghaninne) [inline lib/4_filter_numbers.js]: Can you explain what the `map` function does and why you included this line?

Comment (noto-alex-j) [inline lib/4_filter_numbers.js]: Honestly I got a little help on this one from an example on an MDN page, but their example didn't have a .map in it. The function within the filter specifies filtering any value less than 10 and returns the results as an array, so it was redundant to include the .map iteration. In fact, I have no idea why I did that or where I got it from. It works without it, so I removed it.

**Comment by noto-alex-j** [comment]
Answered question!

**Comment by halfghaninne** [comment]
Nice! I'd still like to have a conversation about what `arr.map(someExpression)` does (I find the examples in the MDN [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) helpful), and why `.map(function(x) {return x})` in particular did not meaningfully change your filtered array.

**Comment by noto-alex-j** [comment]
Of course! From what I've learned, the .map function iterates over an array and then returns the results of the iteration, as opposed to returning the original array a la .each. The .map function I used didn't meaningfully change my filtered array because it literally just returned the same array that was fed into it, which was in turn assigned to the variable newarr that I defined, which was then returned to the filter_numbers function. Without my .map, the array was still assigned to the variable newarr, so it was redundant. I guess I wanted my code to be super explicit ;)

**Pull request closed without merging**